### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by dandick** [johngodley/redirection#1734]
```
Plugin: 4.2.3
WordPress: 5.1.1 (multi)
PHP: 7.3.2
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36
JavaScript: http://danieljdick.com/wp-content/plugins/redirection/redirection.js
REST API: http://danieljdick.com/wp-json/

Error: Failed to fetch
```

**Comment by johngodley** [comment]
Your site is redirecting from http => https. Please update your Settings > General URLs as indicated in the message shown in Redirection.

**Comment by dandick** [comment]
Unfortunately, I followed those instructions, went to Settings > General, and did not find anything on that page.  Are these instructions applicable to multi-site wordpress?  Are there any modules that would remove the possibility of choosing whether the site is to emit an http or https in its links?  Also, I'm running Nginx.  Another consideration is that I am running it in a docker container and am starting it with docker-compose but also have an outer Nginx session on the main system.  In other words, the main VM has Nginx configured to forward traffic to the website to a port on the Nginx container running PHP 7 and PHP-FPM.  But it would seem that as long as the whole set-up can receive requests with a url containing https or coming through port 443, or have it redirect from 80 to 443 in nginx, then when it gets to Wordpress, it should process the URL properly.

But I'm not seeing anything in Settings > General as described in the instructions for fixing this problem.  I'm seeing a selection for a url relative to the hostname and one for a url without the hostname beginning at the root and a relative url--a radio selection basically.  But I am not seeing the http or https.  I did not see that under the Domans tabs either under the network Dashboard either.
